codingStandards/CodingStandard-Html.md

```diff
@@ -237,3 +237,34 @@
       </td>
     </tr>
   </table>
+
+- Always remember to include `rel="noopener noreferrer"` when using `target="_blank"` in links. 
+> Leaving this out would enable attackers to get hold of `window.opener` object, which can be used to redirect to a malicious link. [Source](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c)
```

Comment:
@acjh No it is not possible to have the brackets without escaping. However, I've looked at other markdown files and all other links in the other documentations have no brackets, so I've just removed them all together.